com.github.tenderowl.norka.json

```diff
@@ -0,0 +1,92 @@
+{
+    "app-id": "com.github.tenderowl.norka",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.tenderowl.norka",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:rw",
+        "--own-name=com.github.tenderowl.norka"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
```

Comment:
Any reason for not using the elementary baseApp?

Reply:
It's Python-based and not requires all of the modules in baseApp. Actually I try to make it usable for GNOME too.

Reply:
Sure thing